app/scripts/script.js

```diff
@@ -0,0 +1,69 @@
+// DateTime Dsplay
+function startTime() {
+    var today = new Date();
+    document.getElementById('datetime').innerHTML =
+     today;
+    var t = setTimeout(startTime, 500);
+  }
+  function checkTime(i) {
+    if (i < 10) {i = "0" + i};  // add zero in front of numbers < 10
+    return i;
+  }
```

Comment:
Don't be afraid to let functions take up multiple lines
Multiline functions are easier to read
```suggestion
  // add zero in front of numbers < 10
  function checkTime(i) {
    if (i < 10) {
      return "0" + i
    } else {
      return i;
    }
  }
```